packages/openfeature-server-provider/src/ConfidenceServerProvider.ts

```diff
@@ -137,6 +68,7 @@ export class ConfidenceServerProvider implements Provider {
     context: EvaluationContext,
     logger: Logger,
   ): Promise<ResolutionDetails<T>> {
+    Value.assertValue(defaultValue);
```

Comment:
Should this be part of the other `resolveXEvaluation`?

Reply:
Only needed in `resolveXEvaluation()` where it is not evident that the defaultValue type can "fit" in `Value`.